### PR TITLE
Package the language server for publishing in NPM

### PR DESCRIPTION
**Pull request by bugarela**
Hello :octocat: 

I want to publish our language server to NPM in order to make it easy to install and run from the LSP client for Emacs.

This shouldn't change anything about how we publish the VSCode extension, at least for now. I'm also not automating the release process yet. I plan to run a manual release once this PR is approved.

- [ ] Tests added for any new code
- [ ] Documentation added for any new functionality
- [ ] Entries added to the respective `CHANGELOG.md` for any new functionality
- [ ] Feature table on [`README.md`](../README.md#roadmap) updated for any listed functionality

### CODE REVIEW

File: vscode/quint-vscode/server/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Quint Language Server
+
+[![npm (scoped)](https://img.shields.io/npm/v/@informalsystems/quint-language-server)](https://www.npmjs.com/package/@informalsystems/quint-language-server)
```
Comment (konnov) [inline vscode/quint-vscode/server/README.md:3]: This link looks like two links?

**Review by konnov — APPROVED** [review]
Looks good! I wonder why the diff size is 12k?

File: vscode/quint-vscode/server/README.md (hunk 1 above)
Comment (bugarela) [reply vscode/quint-vscode/server/README.md:3]: It's the badge thing, I believe the first one gets the image and the second is the actual link that opens when you click the badge. I just copied from quint's readme and changed the links.

**Comment by bugarela** [comment]
> Looks good! I wonder why the diff size is 12k?

It's only the `package.lock` file causing that huge diff. Idk why it changed so much, but since I renamed the package itself (from `quint-lsp-server` to `quint-language-server`, it might have affected the hashes of every single dependency or something like that.

**Comment by thpani** [comment]
> It's only the `package.lock` file causing that huge diff. Idk why it changed so much

Looks like indentation changed from 8 to 2 blanks 😄

**Pull request merged**